### PR TITLE
Task3.2

### PR DESCRIPTION
**Pull request by Nikavar**
finished task3.2 _ add comment to the game

### CODE REVIEW

File: GameStore/Controllers/GameController.cs [hunk 1]
```diff
@@ -162,12 +162,21 @@ public async Task<ActionResult> DeleteAsync([FromBody] GameModel model)
...
         [HttpGet("GetCommentsByGameId")]
         public async Task<ActionResult> GetCommentsByGameIdAsync([FromQuery] int? gameId)
         {
             var model = await commentService.GetCommentsByGameIdAsync(gameId);
             return Ok(model);
         }
+
+        // task 3.2
+        [HttpPost("PostComment")]
+        public async Task<ActionResult> PostCommentToGameAsync(CommentModel model)
```
Comment (ggabunia) [inline GameStore/Controllers/GameController.cs:179]: Same as previous task: new controller, gert gameID from route (not model  body directly) and assign that value to the model.gameId, then call create method

File: GameStore.Service/Interfaces/ICommentService.cs [hunk 2]
```diff
@@ -11,5 +11,6 @@ namespace GameStore.Service.Interfaces
     public interface ICommentService
     {
         Task<IEnumerable<CommentModel>> GetCommentsByGameIdAsync(int? id);
+        Task<CommentModel> PostCommentToGameAsync(CommentModel model);
```
Comment (ggabunia) [inline GameStore.Service/Interfaces/ICommentService.cs:14]: just call it AddCommentAsync, or something like that.

File: GameStore/Controllers/GameController.cs (hunk 1 above)
Comment (Nikavar) [reply GameStore/Controllers/GameController.cs:179]: [HttpPost("Game/{gameId}/AddComment")]
        public async Task<ActionResult> AddCommentToGameAsync([FromRoute] int gameId, CommentModel model)
        {
            model.Id = gameId;
            var result = await commentService.AddCommentAsync(model);
            return Ok(result);
        }

File: GameStore.Service/Interfaces/ICommentService.cs (hunk 2 above)
Comment (Nikavar) [reply GameStore.Service/Interfaces/ICommentService.cs:14]: public interface ICommentService
    {
        Task<CommentModel> AddCommentAsync(CommentModel model);
    }

**Pull request merged**